image/image_linux.go

```diff
@@ -474,7 +474,10 @@ func (s *imageSeeder) deriveInfoForLocalSnaps(f seedwriter.SeedAssertionFetcher,
 			return nil, err
 		}
 
-		if err := s.w.SetInfo(sn, info); err != nil {
+		// For local snaps, the component information is set inside
+		// w.SetInfo by looking at the local components information set
+		// in the call to w.SetOptionsSnaps.
```

Comment:
I still think we likely have to change this approach once we will derive side info stuff also for components as well, I think we should put a TODO somewhere, maybe even here about revisiting this

Reply:
TODO added